client/src/components/FormAreasOfFocus/index.js

```diff
@@ -0,0 +1,100 @@
+import React, { useState } from "react";
+
+const FormAreasOfFocus = () => {
+
+	const [areasOfFocus, setAreasOfFocus] = useState([
+		{
+			toWorkOn: [""],
+			okayAt: [""],
+			goodAt: [""],
+		},
+	]);
+
+	const [areaForm, setAreaForm] = useState({
+		message: "",
+		light: "",
+	});
+
+	const handleSubmit = (e) => {
+		e.preventDefault();
+		e.target.reset();
+	};
+
+
+	return (
+		<form onSubmit={handleSubmit}>
+			<div>
+				<p className="areas-input-label">
+            Add area of focus:
+				</p>
+			</div>
+			<div className="input-lights-add-button-container">
+				<div className="area-input-section">
+					<input
+						className="areas-input"
+						type="text"
+						name="area"
+						placeholder="eg. CSS, useEffect Hook, for loop..."
+						onChange={(e) =>
+							setAreaForm({
+								...areaForm,
```

Comment:
For this setState function as well as the following two, please push the changes from today's call before the merge.